libs/langchain-openai/src/chat_models.ts

```diff
@@ -1006,7 +1003,7 @@ export class ChatOpenAI<
     CallOptions extends ChatOpenAICallOptions = ChatOpenAICallOptions
   >
   extends BaseChatModel<CallOptions, AIMessageChunk>
-  implements OpenAIChatInput, AzureOpenAIInput
+  implements Partial<OpenAIChatInput>
```

Comment:
was this a bug before or is there something we're doing to not require all inputs anymore

Reply:
To not require all inputs